data/weapons/weapon_data.csv

```diff
@@ -1,59 +1,59 @@
 name,id,tier,rarity,base value,range,damage/second,damage/shot,emp,impact,turn rate,OPs,ammo,ammo/sec,reload size,type,energy/shot,energy/second,chargeup,chargedown,burst size,burst delay,min spread,max spread,spread/shot,spread decay/sec,beam speed,proj speed,launch speed,flight time,proj hitpoints,autofireAccBonus,extraArcForAI,hints,tags,groupTag,tech/manufacturer,for weapon tooltip>>,primaryRoleStr,speedStr,trackingStr,turnRateStr,accuracyStr,customPrimary,customPrimaryHL,customAncillary,customAncillaryHL,noDPSInTooltip,number
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-IBIS,diableavionics_ibis,0,,300,350,,25,,,90,4,24,10,8,FRAGMENTATION,3,,0,0.1,,,0,2,0.2,1,,1000,,,10,,,PD,"pd6, SR, diable_weapon_bp",,Diable Avionics,,Point Defense,,,,,,,,,,1.1
-Raptor Defense System,diableavionics_raptor,0,,1000,600,,40,,,75,8,48,10,4,FRAGMENTATION,10,,0,0.1,,,0,2,0.2,1,,1000,,,10,,,PD,"pd11, diable_weapon_bp",,Diable Avionics,,Point Defense,,,,,,,,,,
+IBIS,diableavionics_ibis,0,,300,350,,25,,,180,4,24,10,8,FRAGMENTATION,3,,0,0.05,,,0,2,0.2,1,,2000,,,10,,,PD,"pd6, SR, diable_weapon_bp",,Diable Avionics,,Point Defense,,,,,,,,,,1.1
+Raptor Defense System,diableavionics_raptor,0,,1000,600,,40,,,75,8,64,10,4,FRAGMENTATION,10,,0,0.05,,,0,2,0.2,1,,2000,,,10,,,PD,"pd11, diable_weapon_bp",,Diable Avionics,,Point Defense,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Recson S,diableavionics_recson,1,,700,800,,200,,,24,8,,,,KINETIC,225,,0,2,1,,2,6,5,6,,650,,,100,,,,"kinetic9, LR, diable_weapon_bp",,Diable Avionics,,Anti Shield,,,,,,,,,,
+Recson S,diableavionics_recson,1,,700,800,,250,,,24,8,,,,KINETIC,275,,0,1.8,1,,2,6,5,6,,650,,,100,,,,"kinetic9, LR, diable_weapon_bp",,Diable Avionics,,Anti Shield,,,,,,,,,,
```

Comment:
Recson S now does 50 more kinetic damage. it also consumes 50 more flux per shot and fires 10% faster.